Docs: Move hcp_packer_registry block out of the build block (#13427)
hashicorp/packer: website/content/docs/templates/hcl_templates/blocks/build/hcp_packer_registry.mdx

This block is available from version 1.7.7 of Packer.

> The presence of a `hcp_packer_registry` block in a template file will enable HCP
> Packer mode. Packer will push all builds within that template to the remote

Reviewer: ```suggestion
Add a `hcp_packer_registry` block to your template file to enable HCP
Packer mode. When in HCP Packer mode, Packer pushes all builds in the template to the remote
```

> Packer mode. Packer will push all builds within that template to the remote

Reviewer 2: @nphilbrook here by the build do you mean images or just the metadata, asking this to understand if we store just the metadata or images too.
